eng/pipelines/templates/jobs/archetype-sdk-client.yml

```
@@ -447,13 +447,13 @@ jobs:
 
     - task: CopyFiles@2
       inputs:
-        sourceFolder: docs/html
+        sourceFolder: doc/docs/html
```

Same here/elsewhere.
```suggestion
        sourceFolder: doc/generated_doxygen_docs/html
```